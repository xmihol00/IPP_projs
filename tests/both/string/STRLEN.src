.IPPcode21

DEFVAR GF@len
STRLEN GF@len string@ahoj
WRITE GF@len
WRITE string@\010

DEFVAR GF@string 
MOVE GF@string string@string
STRLEN GF@len GF@string
WRITE GF@len
WRITE string@\010

STRLEN GF@string GF@string
WRITE GF@string
WRITE string@\010

MOVE GF@string string@string\032:D
WRITE string@\010

CREATEFRAME
DEFVAR TF@len

STRLEN TF@len string@\120\065\898
WRITE TF@len
WRITE string@\010

STRLEN TF@len string@1\120\065\8985
WRITE TF@len
WRITE string@\010

DEFVAR TF@string
MOVE TF@string string@:)
STRLEN TF@len TF@string
WRITE TF@len
WRITE string@\010

STRLEN TF@len GF@string
WRITE TF@len
WRITE string@\010

WRITE string@\010

PUSHFRAME
STRLEN LF@len LF@string
WRITE LF@len
WRITE string@\010

DEFVAR LF@var
MOVE LF@var string@---
STRLEN GF@len LF@var
WRITE GF@len
WRITE string@\010

WRITE string@\010

POPFRAME
STRLEN TF@len TF@string
WRITE TF@len
WRITE string@\010

STRLEN TF@len GF@string
WRITE TF@len
WRITE string@\010

STRLEN GF@len TF@var
WRITE GF@len
WRITE string@\010
