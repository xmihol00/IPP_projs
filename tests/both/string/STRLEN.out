4
6
6

3
5
2
9

2
3

2
9
3
